src/devices/Rtc/Devices/Ds3231/Ds3231Alarm.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Rtc
+{
+    /// <summary>
+    /// Available alarms on the DS3231
+    /// </summary>
+    public enum Ds3231Alarm
```

Comment:
I don't recall the datasheet saying that both alarms cannot be active at the same time, and there are sources saying that they can both be active, however I was unable to get both to work at the same time. Both alarms worked fine independently, but as soon as they were both enabled, neither alarm would trigger. Took quite some time to figure that out.

I can have another look if you'd like, but I remember looking at the raw values of all registers to confirm that everything was correct, and that nothing except the two alarm enabled bits were changing between enabling one alarm and enabling both.

Reply:
That's odd then. Maybe make it a flag anyway and note the above in the comment? The API should reflect the ideal case of how things should work, so that any underlying bugs (be it our implementation or the physical layer itself) can later be fixed without an api change.

Reply:
Apologies for the delayed response. I've been too busy to spend any time on this project but I will definitely get back to you on these two issues when I have the time again.

Reply:
I think this should be internal as well

Reply:
It seems that both alarms can be setup together. But I can't test as I don't have the device. So leaving it like it is.